### PR TITLE
UPDATE: appending logger name convention

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

reference: https://github.com/chaos-mesh/chaos-mesh/pull/2812#discussion_r791399037.

### CODE REVIEW

**Review by Andrewmatilde — APPROVED** [review]
I agree with you.

File: text/2021-12-09-logging.md [hunk 1]
```diff
@@ -209,12 +209,30 @@ func NewWebModule(logger logr.Logger) *WebModule {
 }
 ```
 
+The name of the logger should follow these rules:
+
+- use kebab-case for the name
+- follow pattern: `<application-name>.<component-name>.<component-name>`
+
```
Comment (iguoyr) [inline text/2021-12-09-logging.md:216]: Do we need to be case-sensitive? For example, all lower-case?

Comment (STRRL) [reply text/2021-12-09-logging.md:216]: "kebab-case" requires using lower-cases :)

Comment (iguoyr) [reply text/2021-12-09-logging.md:216]: Get!

**Review by iguoyr — APPROVED** [review]
LGTM

**Pull request merged**